regulations/management/commands/cache_webpages.py

```diff
@@ -0,0 +1,114 @@
+from django.core.management.base import BaseCommand
+from BeautifulSoup import BeautifulSoup
+import urllib2
+from urlparse import urlparse
+import threading
+import sys
+import time
+
+
+class Command(BaseCommand):
+    args = "<url where eregulations is located> <optional- comma seperated list of regulation locations to be parsed >  "
+    help = 'call every page in eregulations allowing the pages to be cached'
+
+    def access_url(obj,*args):
+
+        try:
+            without_anchor = args[0].split("#")[0]
+            temp = urllib2.urlopen(without_anchor)
+            temp.read()
+            temp.close()
+            print "Cached: " + without_anchor
```

Comment:
Note that you should use self.stdout.write() and self.stderr.write() instead of print statements in your Django management commands. See: https://docs.djangoproject.com/en/1.7/howto/custom-management-commands/#module-django.core.management